problems/concat/problem.txt

```diff
@@ -24,3 +24,13 @@ process.stdin.
 
 Make sure to `npm install concat-stream` in the directory where your solution
 file is located.
+
+Hints:
+
+Both process.stdout and concat() are writeable streams, which means they
+can't be piped together. Outlined in the documentation is an alternative
+way to write to process.stdout.
```

Comment:
I'd suggest that the console.log() method is also included in the hint or a link to [the docs](https://nodejs.org/api/process.html#process_process_stdout) is given. So far, this is a good hint for questions [like this](https://github.com/workshopper/stream-adventure/issues/133#issue-65135736), it just feels more helpful to have that together with this pull request, rather than having another issues asking for the same questions later.